README.md

```diff
@@ -30,145 +30,144 @@ Styleguide for low-fidelity paper prototyping and wireframing.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/53.PNG)
 
-5.4. _MUST_. Mark elements and controls that contains or triggers AJAX loaders with an horizontal three-dot.
+5.4. _MUST_ use horizontal ellipsis (`...`) for elements or controls that contain or trigger AJAX loaders.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/54.PNG)
 
-5.5. _MUST_. Mark elements and controls that are repeatable by some iteration with a vertical three-dot. Indicate more details of the iteration in the Table of Features.
+5.5. _MUST_ use vertical ellipsis (`⋮`) for elements or controls that repeat by some iteration with a vertical three-dot. Indicate more details of the iteration in the Table of Features.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/55.PNG)
 
 ### 6. Green elements 💚 Controls
 
-6.1. _MUST_. Draw **input/text boxes** with just an horizontal line. DO NOT draw inputs as a rectangle, as they may be confused with a button.
+6.1. _MUST_ draw **input/text boxes** with just a horizontal line. DO NOT draw inputs as rectangles, as they may be confused with buttons.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/61.PNG)
 
-6.2. _MUST_. Draw **buttons** as rectangles, including **image buttons**. For buttons that have a cancel/abort/close behavior include a times character (`×`) aligned to center.
+6.2. _MUST_ draw **buttons** as rectangles, including **image buttons**. For buttons that have a cancel/abort/close behavior include a times character (`×`) aligned to its center.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/62.PNG)
 
-6.3. _MUST_. Draw **checkboxes** as squares with a checkmark inside.
+6.3. _MUST_ draw **checkboxes** as squares with a chec kmark inside (`☑`).
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/63.PNG)
 
-6.4. _MUST_. Draw **radio buttons** as circles with a dot inside.
+6.4. _MUST_ draw **radio buttons** as circles with a dot inside (`🔘`).
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/64.PNG)
 
-6.5. _MUST_. Draw **dropdown menus** writing the "Dropdown" word, followed by a feature reference code (as described in 4.2), and, followed by a black triangle ponting down (`▼`). 
+6.5. _MUST_ represent **dropdown menus** writing "Dropdown", followed by a feature reference code (as described by 4.2) and a black triangle ponting down (`▼`).
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/65.PNG)
 
-6.5.1. _MUST_. For the menu panel, draw it as a black container (as described in 3.1), but make a diagonal (northwest to souteast) line at the top-right corner of the container. 
+6.5.1. _MUST_ draw black containers (as described in 3.1) with a diagonal line from the top to the right line of the container's top-right corner for panel menus.
```

Comment:
"Panel menus" doesn't seem to be connected with dropdowns, like I wrote before. In this new writing, I think they can be defined as "dropdown menus".

Reply:
Actually, neither do